frontend/public/components/container.jsx

```diff
@@ -110,7 +110,10 @@ const Env = ({env}) => {
 
 const Details = (props) => {
   const pod = props.obj;
```

Comment:
This could even be:

```JavaScript 
const container = _.find(pod.spec.containers, {name: props.match.params.name}) || 
                  _.find(pod.spec.initContainers, {name: props.match.params.name});
if (!container) {
  return null;	    
}
```
You can use the "or" operator `||` and eliminate the duplicate `if`.  The second `_.find()` won't run if the first `_.find()` returns an item.